x-pack/legacy/plugins/maps/public/components/map/feature_properties.test.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { FeatureProperties } from './feature_properties';
+
+class MockTooltipProperty {
+  constructor(key, value, isFilterable) {
+    this._key = key;
+    this._value = value;
+    this._isFilterable = isFilterable;
+  }
+
+  isFilterable() {
+    return this._isFilterable;
+  }
+
+  getFilterAction() {
+    return () => {};
+  }
+
+  getHtmlDisplayValue() {
+    return this._value;
+  }
+
+  getPropertyName() {
+    return this._key;
+  }
+}
+
+const defaultProps = {
+  loadFeatureProperties: () => { return []; },
+  featureId: `feature`,
+  layerId: `layer`,
+  onCloseTooltip: () => {},
+  showFilterButtons: false
+};
+
+const mockTooltipProperties = [
+  new MockTooltipProperty('foo', 'bar', true),
+  new MockTooltipProperty('foo', 'bar', false)
```

Comment:
Can you change the name/value of the props so its easier to trace in the snapshots? May name one `prop1` and `prop2`?